CallAutomation_CallLiveTranscription/readme.md

```diff
@@ -0,0 +1,65 @@
+﻿|page_type|languages|products
+|---|---|---|
+|sample|<table><tr><td>csharp</tr></td></table>|<table><tr><td>azure</td><td>azure-communication-services</td></tr></table>|
+
+# Call Live Transcription - Quick Start Sample
+
+This sample application shows how the Azure Communication Services  - Call Automation SDK can be used generate the live transcription between PSTN calls. 
+It accepts an incoming call from a phone number, performs DTMF recognition, and transfer the call to agent. You can see the live transcription in websocket during the conversation between agent and user. The application is a web-based application built on .Net7 framework.
+
+## Prerequisites
+
+- Create an Azure account with an active subscription. For details, see [Create an account for free](https://azure.microsoft.com/free/)
+- Create an Azure Communication Services resource. For details, see [Create an Azure Communication Resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource). You will need to record your resource **connection string** for this sample.
+- Get a phone number for your new Azure Communication Services resource. For details, see [Get a phone number](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/telephony/get-phone-number?tabs=windows&pivots=programming-language-csharp)
+- Create Azure AI Multi Service resource. For details, see [Create an Azure AI Multi service](https://learn.microsoft.com/en-us/azure/cognitive-services/cognitive-services-apis-create-account).
+- Install ngrok. Instructions [here](https://ngrok.com/)
+- Setup websocket
```

Comment:
this just says setup websocket, is the scope of this sample app to provide a websocket sample as well ?

Reply:
In that way we could also demonstrate the transcription metadata and data parser from the SDK

Reply:
Im not sure whether we wanted to maintain the websocket server code also, tagging @bharat-kalyan-namburi for his opinion